app/jobs/dradis/plugins/csv/mapping_import_job.rb

```diff
@@ -0,0 +1,82 @@
+require 'csv'
+
+module Dradis::Plugins::CSV
+  class MappingImportJob < ApplicationJob
+    queue_as :dradis_project
+
+    def perform(file:, identifier_col_index:, mappings:, project_id:, uid:)
```

Comment:
Can you add in a comment here what is the expected mappings format?

Reply:
Done